frontend/src/composables/useChainSelection.ts

```diff
@@ -27,7 +27,7 @@ export function useChainSelection(
     if (provider.value && chain.identifier !== provider.value.chainId.value) {
       try {
         const isSuccessfulSwitch = await provider.value.switchChain(chain.identifier);
-        if (isSuccessfulSwitch === null) {
+        if (isSuccessfulSwitch === false) {
```

Comment:
Wouldn't it be better to write `if (!isSuccessfulSwitch)`?

Reply:
yes, definitely better! thx